prow/pubsub/subscriber/subscriber.go

```diff
@@ -161,25 +170,36 @@ func (s *Subscriber) handlePeriodicJob(l *logrus.Entry, msg messageInterface, su
 	if periodicJob == nil {
 		err := fmt.Errorf("failed to find associated periodic job %s", pe.Name)
 		l.WithError(err).Errorf("failed to create job %s", pe.Name)
+		fakePJ := pjutil.NewProwJobWithAnnotation(kube.ProwJobSpec{}, nil, pe.Annotations)
+		if s.Reporter.ShouldReport(&fakePJ) {
+			s.Reporter.Report(&fakePJ)
+		}
```

Comment:
hummm?

Reply:
it probably should send back the error, not a fake prowjob..

Reply:
Yeah so we need to report that the job could not be created somehow. Since the job could not be found we still need to report it. Does it make sense ?

Reply:
I think - when `handlePeriodicJob` errors out, the handler should post a pubsub message back